frontend/src/App.test.js

```diff
@@ -1,8 +0,0 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
-test('renders learn react link', () => {
```

Comment:
we should leave this test in some form it is a good example of how to write unit tests in react